src/OrbitGl/App.cpp

```diff
@@ -254,6 +248,21 @@ OrbitApp::OrbitApp(orbit_gl::MainWindowInterface* main_window,
         action->Execute();
         ORBIT_STOP();
       });
+
+  const unsigned number_of_logical_cores = std::thread::hardware_concurrency();
+  // std::thread::hardware_concurrency may return 0 on unsupported platforms but that shouldn't
+  // occur on standard Linux or Windows.
+  CHECK(number_of_logical_cores > 0);
+
+  core_count_sized_thread_pool_ = ThreadPool::Create(
```

Comment:
The question I have is why you chose to add another `ThreadPool` instead of simply using `std::thread::hardware_concurrency()` in `FunctionsDataView::DoFilter`. Is it really worth adding another `ThreadPool` to `OrbitApp` for this?

Reply:
As I understood it this was a suggestion from Dimitry.

Reply:
Yes I think it is better to have a separate thread-pool here since the OrbitApp's one is not tailored to be used for high-cpu-load tasks. It handles mostly "do not block main thread while waiting on io"

Reply:
I'm gonna close this.